addons/payment/views/payment_templates.xml

```diff
@@ -103,7 +103,7 @@
                     </t>
                 </t>
                 <t t-foreach="pms" t-as="pm">
-                    <div class="card-body">
+                    <div class="card-body o_payment_acquirer">
```

Comment:
o_payment_acquirer
o_delivery_carrier

Could you use more specific name please to avoid potential futur conflict

o_delivery_carrier_select for example